unison-src/transcripts/branch-command.md

```diff
@@ -33,22 +33,17 @@ foo/main> branch /main /topic8
 foo/main> branch /main foo/topic9
 foo/main> branch foo/main topic10
 foo/main> branch foo/main /topic11
-.> branch foo/main foo/topic12
+scratch/main> branch foo/main foo/topic12
 
 foo/main> branch bar/topic
 bar/main> branch foo/main topic2
 bar/main> branch foo/main /topic3
-.> branch foo/main bar/topic4
-
-.some.loose.code> branch foo/topic13
```

Comment:
These tests I just removed since we'll be removing loose code support very soon.